examples/reference-entity-model/steps/mapping/mapCustomersJSON.step.json

```diff
@@ -10,7 +10,7 @@
   "collections": [
     "default-mapping"
   ],
-  "additionalCollections": [],
+  "additionalCollections": ["Customer","mapCustomersJSON"],
```

Comment:
Can you change "collections" to be this array? Check to see if any other step depends on "default-mapping", but that's the right fix here.

Reply:
Moved to collections.